avocado-setup.py

```diff
@@ -712,6 +735,8 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
                                                          Testsuites[test_suite].runsummary))
             summary_output.append(Testsuites[test_suite].runlink)
         logger.info("\n".join(summary_output))
+        if not (len(postscript) == 1 and not postscript[0]):
+            configure_prepostscript(postscript, 'post')
```

Comment:
Why should this be in `parse_test_config()`, and not in `env_clean()` ?

Reply:
yes that is a good idea